draft-ietf-tls-tls13.md

```diff
@@ -1567,8 +1567,8 @@ cipher_suites
   the server does not recognize, support, or wish to use, the server
   MUST ignore those cipher suites, and process the remaining ones as
   usual. Values are defined in {{cipher-suites}}. If the client is
-  attempting a PSK key establishment, it MUST only advertise the single
-  cipher suite associated with the PSK.
+  attempting a PSK key establishment, it MUST advertise at least one
+  cipher suite containing a Hash associated with the PSK.
```

Comment:
> On a client, it actually should be ignored too. Either you keep all your sessions in-memory, in which case you necessarily satisfy this and send the cipher list you would have sent anyway.

Hmm... I think this situation doesn't encompass out-of-bands PSKs, or some other non-web scenarios.